code/game/machinery/station_holomap.dm

```diff
@@ -194,6 +194,48 @@
 
 // TODO: Make these constructable.
 
+/obj/machinery/station_map/robot/Initialize()
+	return
```

Comment:
This needs to return a valid enum value.

Reply:
what do you mean. Both parent and machine return obj's return.